### DISCUSSION TITLE
Include Date Modified into 2sxc API

### DISCUSSION THREAD

**Discussion by Chris-Behling** [2sic/2sxc#2310] [General] [answered]
What would be involved to include the data for Date Modified into the 2sxc API?

I have a project where we'd like to leverage the 2sxc API for some external applications to allow it to consume some data. This data can be time sensitive and having a Last Updated parameter is desired. Typically you could just create an input control for this and allow the user to update, but you can't always rely on the user updating this information when necessary.

I noticed that the History view for 2sxc keeps track of the date updated so I am hopeful here that this data can be easily exposed or included in the API.

In the below example, ID, Guid. and Title are dynamically added to the API feed. Having LastModified and even potentially a 2sxc version ID could be really helpful.

`[{
	"Version": "1.0.0",
	"Content": "<p>Hello World</p>",
	"LastModified": "2020-12-17T00:00:00Z",
	"Id": 3216,
	"Guid": "81fa4909-e111-46f3-a468-f3a8cd504f3a",
	"Title": "1.0.0"
}]`

**Comment by iJungleboy** [comment]
So basically there are a few things to consider. Technically not a big deal, but I'm not sure if we're letting out information which may be a bit sensitive. Let me think about it...

BTW: At least in Razor and WebApi you can get the Modified date - property Modified :).

**Reply by Chris-Behling** [reply]
Wouldn't the security behind the API make that a non-issue? By default the API is only exposed to administrators, and then you can expose it to other user groups, or even non-authenticated users.

**Comment by iJungleboy** [comment, marked as answer]
I think you're right - I'll just add it.
